### ISSUE TITLE
Standardise .README structure

### ISSUE THREAD

**Issue by IljaDaderko** [react-native-community/meta#16]
Looking at various repos, it is clear that README files don't follow common structure. I feel like users would benefit a lot if we define specific sections for README files like `Installation / Example / API / Requirements ...` and order them in same manner. This will mean that anyone familiar with other packages in react native community will be familiar with readme's and able to find what they are looking for faster.

We should also try to stick with same formatting where possible.

**Comment by grabbou** [comment]
I like what Hapi.js community does in this space - they have a lead maintainer - person that is a point of contact re: that module. This could be a changing role on some sort of a cadence. 

Some repositories started having `Contributors` and a list of developers with the link to the company they work for. 

I think this is a good and non-intrusive way to communicate who's working on a given module.

**Comment by kelset** [comment]
yeah I really like this, we should have some sort of defined structure like:
```
NAME OF THE PROJECT

description

how to contribute

license link

maintainers
```

dunnow something like that

**Comment by grabbou** [comment]
Also check the CLI readme structure https://github.com/react-native-community/cli - we have few sections and "usage" too. 

I am not sure if we can enforce the "exact" structure, but maybe a "minimum list of sections" that can be extended if project needs it.

**Comment by pvinis** [comment]
'installation' and 'usage' are useful sections.

**Comment by kelset** [comment]
> I am not sure if we can enforce the "exact" structure, but maybe a "minimum list of sections" that can be extended if project needs it.

yeah absolutely
